sql/moz-fx-data-shared-prod/telemetry_derived/forecasts_cache_v1/init.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE TABLE
+  telemetry_derived.forecasts_cache_v1
```

Comment:
Can we name this something more specific like `looker_kpi_forecast_cache`?

Reply:
I think that's appropriate. I'll update the table